linalg/sundials.hpp

```diff
@@ -62,10 +73,10 @@ using SUNContext = void*;
 namespace mfem
 {
 
-#ifdef MFEM_USE_CUDA
+#if defined(MFEM_USE_CUDA) || defined(MFEM_USE_HIP)
```

Comment:
Might be worth just defining something like `#define SUNDIALS_USE_GPU = MFEM_USE_CUDA || MFEM_USE_HIP` if there will be support for more backends to avoid having to update multiple conditionals and comments?  Or maybe it's not worth worrying about that now

Reply:
When SYCL support is added in the future, it is unclear if we will need to have a distinct code path from CUDA/HIP or not. So I would recommend leaving this as is for now and re-evaluating when SYCL is added.